docs/platform/concepts/plan-upgrades.rst

```diff
@@ -0,0 +1,16 @@
+Prepare for larger service plans
```

Comment:
I believe the title could be more informative? The article seems to be related to billing and credit card details but the title seems to point people to an howto move to a larger plan?

Reply:
Maybe the title should be "Solving issues when upgrading to larger service plans"